src/TestSuite/Constraint/Console/ExitCode.php

```diff
@@ -57,6 +57,6 @@ public function matches($other): bool
      */
     public function toString(): string
     {
-        return sprintf('matches exit code %d', $this->exitCode);
+        return sprintf('matches exit code %s', $this->exitCode ?? 'null');
```

Comment:
Why is this needed if you  have exist code defined and either null or int?
$this->exitCode should suffice then?

Reply:
Nope, cause I want string "null" to be used if exit code is `null`.

Reply:
I see. I didnt see the quotes around it ;) Early for me.